### PR TITLE
[REF] mail: `ThreadIcon`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2843238

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
Hello @alexkuhn 👋 
This PR is ready for review.

cc. @stefanorigano

File: addons/mail/static/src/components/thread_icon/thread_icon.scss [hunk 1]
```diff
@@ -3,24 +3,13 @@
 // ------------------------------------------------------------------
 
 .o_ThreadIcon {
-    display: flex;
     width: 13px;
```
[the remark this answers is not in the document]
Comment (Brieuc-brd) [reply addons/mail/static/src/components/thread_icon/thread_icon.scss]: Good point, I updated the branch. 👍

**Review by alexkuhn — CHANGES_REQUESTED** [review]
There are overrides of `ThreadIcon` template, in `im_livechat` and `hr_holidays`. Some changes must also affect these modules.

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**